### PR TITLE
Make test_beamformer_net.py faster

### PR DESCRIPTION
**Pull request by ShigekiKarita**
Related #2459 

216sec -> 12.76sec

- Decrease model sizes
- Remove duplicated and unused parameters

### CODE REVIEW

**Comment by Emrys365** [comment]
Thank you for fixing this script, and sorry for my late response. I'll also check the script tomorrow.

File: test/espnet2/enh/nets/test_beamformer_net.py [hunk 1]
```diff
@@ -11,30 +11,27 @@
...
-@pytest.mark.parametrize("wlayers", [3])
-@pytest.mark.parametrize("wunits", [8])
-@pytest.mark.parametrize("wprojs", [10])
-@pytest.mark.parametrize("wdropout_rate", [0.0, 0.2])
+@pytest.mark.parametrize("wnet_type", ["lstm"])
+@pytest.mark.parametrize("wlayers", [2])
+@pytest.mark.parametrize("wunits", [2])
+@pytest.mark.parametrize("wprojs", [2])
+@pytest.mark.parametrize("dropout_rate", [0.0, 0.2])
 @pytest.mark.parametrize("taps", [5])
```
Comment (Emrys365) [inline test/espnet2/enh/nets/test_beamformer_net.py:23]: This value in Line 23 can also be reduced:
```
@pytest.mark.parametrize("taps", [2])
```

File: test/espnet2/enh/nets/test_beamformer_net.py [hunk 2]
```diff
@@ -111,28 +105,27 @@ def test_beamformer_net_forward_backward(
...
```
Comment (Emrys365) [inline test/espnet2/enh/nets/test_beamformer_net.py:117]: This value can also be reduced:
```
@pytest.mark.parametrize("taps", [2])
```

This comment also applies to Line 219:
```
        taps=2,
```

File: test/espnet2/enh/nets/test_beamformer_net.py [hunk 3]
```diff
@@ -187,7 +179,7 @@ def test_beamformer_net_consistency(
         ref_channel=ref_channel,
         use_noise_mask=use_noise_mask,
         beamformer_type=beamformer_type,
-        bdropout_rate=bdropout_rate,
+        bdropout_rate=dropout_rate,
     )
```
Comment (Emrys365) [inline test/espnet2/enh/nets/test_beamformer_net.py:184]: Additional suggestions:

Line 205:
```
@pytest.mark.parametrize("ch", [1, 2])
```

Line 236:
```
    ch = 3
```

File: test/espnet2/enh/nets/test_beamformer_net.py [hunk 4]
```diff
@@ -111,28 +105,27 @@ def test_beamformer_net_forward_backward(
...
-@pytest.mark.parametrize("wnet_type", ["blstmp"])
-@pytest.mark.parametrize("wlayers", [3])
-@pytest.mark.parametrize("wunits", [8])
-@pytest.mark.parametrize("wprojs", [10])
-@pytest.mark.parametrize("wdropout_rate", [0.0, 0.2])
+@pytest.mark.parametrize("wnet_type", ["lstm"])
+@pytest.mark.parametrize("wlayers", [2])
+@pytest.mark.parametrize("wunits", [2])
+@pytest.mark.parametrize("wprojs", [2])
+@pytest.mark.parametrize("dropout_rate", [0.0, 0.2])
```
Comment (Emrys365) [inline test/espnet2/enh/nets/test_beamformer_net.py:116]: The dropout_rate is useless with `model.eval()`, so Line 116 can be modified to:
```
@pytest.mark.parametrize("dropout_rate", [0.0])
```

**Review by Emrys365 — COMMENTED** [review]
Thank you for the fix. I have the following comments.
With the following modification, the testing time in my local machine decreases to about 9.7sec.

**Comment by ShigekiKarita** [comment]
Thanks, done!

**Pull request merged**